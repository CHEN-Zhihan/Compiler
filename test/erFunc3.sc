def main() {
    return 5;
}

return main();