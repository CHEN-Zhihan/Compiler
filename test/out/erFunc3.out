return occurs in the global scope
